src/monitor-mint-calls.js

```diff
@@ -0,0 +1,79 @@
+const ethers = require('ethers');
+
+const {
+  Finding, FindingSeverity, FindingType,
+} = require('forta-agent');
+
+// load required shared types
+const {
+  VotingToken: votingTokenAddress,
+  Voting: votingAddress,
+} = require('../contract-addresses.json');
+
+const { abi: votingTokenAbi } = require('../abi/VotingToken.json');
+const { umaEverestId } = require('../agent-config.json');
```

Comment:
new NPM abi method might wanna switch it up before merge

Reply:
Added import of `@uma/contracts-node` module and associated function calls.